src/app/games/epaminondas/EpaminondasRules.ts

```diff
@@ -120,9 +153,19 @@ export class EpaminondasRules extends Rules<EpaminondasMove, EpaminondasState, E
         }
         return MGPFallible.success(board);
     }
+
+    private constructor() {
+        super(EpaminondasState, EpaminondasRules.RULES_CONFIG_DESCRIPTION.getDefaultConfig().config);
```

Comment:
`this.getRulesConfigDescription().getDefaultConfig().config` serait plus court et générique

Reply:
out of date